recipes/genozip/meta.yaml

```diff
@@ -0,0 +1,70 @@
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: genozip
+  version: 1.0.1
+
+source:
+  url: https://github.com/divonlan/genozip/raw/master/conda/genozip-1.0.1.tar.gz
+  sha256: 4828aa3ecb400d0bcf5e554e6fb7bc2be01b104f8b7fc81c63924357ef24aacf
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+  #  - make  # [unix]
+    - zlib
```

Comment:
This should only be in `host` and `run`:
```suggestion
```

Reply:
done